examples/siro_sandbox/app_states/app_state_tutorial.py

```diff
@@ -14,32 +14,30 @@ class AppStateTutorial(AppState):
     def __init__(
```

Comment:
Could we add a description about role of this class? Something like actual tutorial containing stages/phases such as A,B,C is defined in hitl_tutorial. This class serves as a way to convert tutorial into an app state that can then be transitioned to from within another app such as the rearrange app..

Reply:
I'll let @0mdc tackle this. [Asana item](https://app.asana.com/0/1205829436198608/1206329593074720)